partials/_hero.scss

```diff
@@ -0,0 +1,50 @@
+.hero {
+    display: flex;
+    align-items: center;
+    width: 100%;
+    position: relative;
+
+    &__img {
+        width: 100%;
+        height: auto;
```

Comment:
<img width="261" alt="Zrzut ekranu 2022-12-1 o 14 13 51" src="https://user-images.githubusercontent.com/87694556/205068205-b74731ff-c583-48b6-a44e-4aa0e69832d1.png">
on mobile the height of the section should change (screen from figma)

Reply:
Struggling getting the image to be centered - it's only showing the first part of the image from the left. align-items/content - center doesn't work neither does background position and size etc. Wondering if I am just getting that all wrong and should be doing something else?

Reply:
can you push your most recent changes to this? Let me know if you figured it out eventually or if you need some help.

Reply:
I think I have now sorted it! :)